Source/CarthageKit/GitHub.swift

```diff
@@ -97,7 +97,7 @@ public struct GitHubRepository: Equatable {
 }
 
 public func ==(lhs: GitHubRepository, rhs: GitHubRepository) -> Bool {
-	return lhs.owner == rhs.owner && lhs.name == rhs.name
+	return lhs.owner.caseInsensitiveCompare(rhs.owner) == NSComparisonResult.OrderedSame && lhs.name.caseInsensitiveCompare(rhs.name) == NSComparisonResult.OrderedSame
```

Comment:
Could you remove the `NSComparisonResult` here so it's just `== .OrderedSame`? :pray: